tensorboard/plugins/projector/vz_projector/vz-projector-inspector-panel.ts

```diff
@@ -25,9 +25,18 @@ namespace vz_projector {
       metadataColumn: String,
       numNN: {type: Number, value: 100},
       updateNumNN: Object,
+      spriteMetadata: Object,
+      showNeighborImages: Boolean,
     },
   });
 
+  export type spriteMetadata = {
```

Comment:
Can we keep this type unexported if possible?

Reply:
oops that was a copy/paste oversight